docs/en/ingest-management/release-notes/release-notes-7.13.asciidoc

```diff
@@ -67,6 +72,24 @@ of {agent}s in {kib}.
 *Impact* +
 If the role and user were set up in a previous release, remove them now to avoid
 them being orphaned in the cluster. {kib-pull}98745[#98745]
+====
+
+[[known-issue-25773]]
+.{agent}s unenrolling from a self-managed {fleet-server} hang at "Updating" and API keys are not invalidated
+[%collapsible]
+====
+
+*Details* 
+
+In {kib}, when you unenroll an {agent} from a self-managed {fleet-server},
+the status may hang at "Updating". This problem only occurs with {agent}s
+enrolled with a service token.
```

Comment:
This problem only occurs with Elastic Agents connecting to a `fleet-server` started with a service token.

Reply:
Fixed. I dropped some detail there. :-)